azkeys/coseSigner.go

```diff
@@ -61,26 +111,18 @@ func base64BEtoBigInt(in string) (*big.Int, error) {
 
 // PublicKey gets the latest key's public key
```

Comment:
This comment is a bit misleading now, it is the one and only key, not the latest necessarily